spec/system/candidate_interface/candidate_selecting_a_course_spec.rb

```diff
@@ -32,8 +32,6 @@
     when_i_click_on_add_another_course
     and_i_choose_that_i_know_where_i_want_to_apply
     and_i_choose_another_provider
-    and_i_choose_another_course
-    then_i_see_the_not_ready_for_you_yet_page
 
     when_i_click_back
```

Comment:
Does clicking back still make sense here? (having not selected a non-applyable course)

Reply:
Good spot!